windows-agent/internal/proservices/landscape/connection.go

```diff
@@ -118,11 +116,14 @@ func newConnection(ctx context.Context, d serviceData) (conn *connection, err er
 // If this is the first connection ever, the server will respond by assigning
 // the host a UID. This Recv is handled by receiveCommands, but handshake
 // waits until the UID is received before returning.
-func handshake(ctx context.Context, d serviceData, conn *connection) error {
+func handshake(ctx context.Context, d serviceData, conn *connection) (err error) {
+	defer decorate.OnError(&err, "could not complete handshake")
+	log.Debugf(ctx, "Landscape: starting handshake")
```

Comment:
`Debugf` but no parameter.

Reply:
I guess that linters don't notice this because it is not a `fmt` directive. Good catch.